composer.json

```diff
@@ -22,11 +22,12 @@
         ]
     },
     "require": {
-        "php": "^5.6||^7.0",
+        "php": "7.2.*",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/doctrine-cache-bundle": "^1.2",
         "doctrine/doctrine-migrations-bundle": "^1.0",
-        "doctrine/orm": "^2.5",
+        "doctrine/orm": "~2.5.0",
+        "doctrine/dbal": "~2.5.12",
```

Comment:
Why us this needed, shouldn't this be a dependency of `doctrine/orm` itself?

Reply:
DBAL was pinned to a 2.5 version as newer versions remain incompatible with our test setup.